plugins/coverbrowser.koplugin/mosaicmenu.lua

```diff
@@ -37,14 +38,9 @@ local BookInfoManager = require("bookinfomanager")
 
 -- We will show a rotated dogear at bottom right corner of cover widget for
 -- opened files (the dogear will make it look like a "used book")
--- The ImageWidget Will be created when we know the available height (and
+-- The ImageWidget will be created when we know the available height (and
 -- recreated if height changes)
-local corner_mark_size = -1
-local corner_mark
-local reading_mark
-local abandoned_mark
-local complete_mark
-local progress_widget
+local corner_mark_size, corner_mark, reading_mark, abandoned_mark, complete_mark, collection_mark, progress_widget
```

Comment:
The way it is right now might lead to nicer diffs in the future? But should be fine.

Reply:
Yes, I also prefer them multilines, at least for this case here.
Here, they are not just little local variables used for some internal work in a function, they are module locals and hold a long-time-alive widget, so it feels clearer to list them one by line, so we really know what's kept/available.